ext/ReactantMPIExt/Overrides.jl

```diff
@@ -0,0 +1,110 @@
+@reactant_overlay @noinline function MPI.Init(; kwargs...)
+    if !isempty(kwargs)
+        @warn "Ignoring MPI.Init kwargs when tracing over MPI..." kwargs...
+    end
+    return Ops.init()
+end
+
+@reactant_overlay @noinline function MPI.Init(; kwargs...)
```

Comment:
This should be Finalize?

```suggestion
@reactant_overlay @noinline function MPI.Finalize(; kwargs...)
```

Reply:
ah funk yeah